docs/user/package-management/history-and-rollback.md

```diff
@@ -19,6 +19,13 @@ eopkg history
 
 ## Rollback
 
+:::warning
```

Comment:
```suggestion
:::info
```
I think this should just be "info". "Warning" would be reserved for a command that would cause damage.

I also note our guidance on admonitions does not match the actual admonitions available with docusaurus. Oops.